### PR TITLE
Convert to helm thanos merge master

### PR DESCRIPTION
**Pull request by ramius345**
Thanos support to address Issue #23 long term storage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,32 @@ Once you are finished adding your extra hosts, you can update your stack by re-r
...
+
+And then:
+
+```
+./runhelm.sh --values file_with_bucket_config.yaml
+```
+
+The above configuration is the default access endpoint when noobaa is installed into the noobaa namespace.  For installaing noobaa, see:
+
+https://github.com/noobaa/noobaa-core#deploy-to-kubernetes
```
Comment (etsauer) [inline README.md:90]: I think we want to be a bit more opinionated than just sending the link to the nooba repo. For example, we don't want people installing this in the `default` namespace, which `noobaa install` does, and they don't provide clear directions on how to create buckets.

File: charts/deploy/templates/thanos-objstore-config-secret.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{{- define "pelorus.thanos.objstore.config" }}
+type: s3
+config:
+  bucket: thanos
```
Comment (etsauer) [inline charts/deploy/templates/thanos-objstore-config-secret.yml:4]: we probably don't want to hardcode this, but allow (and instruct) the user to set a bucket name

File: runhelm.sh [hunk 3]
```diff
@@ -25,6 +27,12 @@ helm template \
     --set openshift_prometheus_basic_auth_pass=$GRAFANA_DATASOURCE_PASSWORD \
     $EXTRA_VALUES \
     ./charts/deploy/ | oc apply -f - -n pelorus
+
+
+#    --set bucket_access_point=$INTERNAL_S3_ENDPOINT \
```
Comment (etsauer) [inline runhelm.sh:32]: did you intend to leave this here?

File: README.md [hunk 4]
```diff
@@ -63,6 +63,32 @@ Once you are finished adding your extra hosts, you can update your stack by re-r
...
+
```
Comment (etsauer) [inline README.md:91]: How does the user verify that this setup is working, and data is persisting?

Comment (ramius345) [reply README.md:101]: I've fixed a few of these defects in the documentation as you've asked, but I've run out of time this weekend.  This basically is going to require a full writeup on how to use different S3 providers which may be its own backlog item.  I've removed this line in the meantime and am deferring to how each S3 provider documents it themselves.  I will revisit it when I have some more time.

File: runhelm.sh (hunk 3 above)
Comment (ramius345) [reply runhelm.sh]: Removed.

File: charts/deploy/templates/thanos-objstore-config-secret.yml (hunk 2 above)
Comment (ramius345) [reply charts/deploy/templates/thanos-objstore-config-secret.yml:7]: I've changed this to the default and provided some documentation on the variable I've added.

File: README.md (hunk 1 above)
Comment (ramius345) [reply README.md]: I didn't have time to write up a full set of documentation on how noobaa works so I have removed this.  I will revisit it when I have more time.  For now the answer is going to be consult your bucket provider of choice' documentation.

File: README.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ Before deploying the tooling, you must have the following prepared
   * The OpenShift Command Line Tool (oc)
   * Helm 3
 
-### Deployment Instructions
+First, we must collect some information from the cluster to feed to our templates.
```
[the remark this answers is not in the document]
Comment (ramius345) [reply README.md]: Removed.

**Comment by etsauer** [comment]
@ramius345 I think since this has been sitting out for a while, we'll just merge and do a followup PR to address these.

**Pull request merged**